features/base/media/reducer.js

```diff
@@ -0,0 +1,112 @@
+import { combineReducers } from 'redux';
+
+import { ReducerRegistry } from '../redux';
+
+import {
+    CAMERA_FACING_MODE_CHANGED,
+    VIDEO_MUTED_STATE_CHANGED,
+    AUDIO_MUTED_STATE_CHANGED
+} from './actionTypes';
+
+import { CAMERA_FACING_MODE } from './constants';
+
+/**
+ * Media state object for local audio.
+ *
+ * @typedef {Object} AudioMediaState
+ * @property {boolean} muted=false - Audio muted state.
+ */
+
+/**
+ * Initial state for local audio.
+ *
+ * @type {AudioMediaState}
+ */
+const AUDIO_INITIAL_MEDIA_STATE = {
+    muted: false
+};
+
+/**
+ * Reducer for audio media state.
+ *
+ * @param {AudioMediaState} state - Media state of local audio.
+ * @param {Object} action - Action object.
+ * @param {Object} action.media - Media state object.
+ * @param {AudioMediaState} action.media.audio - Media state of local audio.
+ * @param {string} action.type - Type of action.
+ * @returns {AudioMediaState}
+ */
+const audio = (state = AUDIO_INITIAL_MEDIA_STATE, action) => {
```

Comment:
That's an anonymous function assigned to a variable. I believe a named function is simpler to read.